validphys2/src/validphys/process_options.py

```diff
@@ -202,14 +202,14 @@ def _displusjet_xq2map(kin_info):
     x = q2 * q2 / s / (pt**2 - q2)
     return x, q2
 
-def _dywboson_xq2map(kin_dict):
+def _dyboson_xq2map(kin_dict):
```

Comment:
```suggestion
def _dyboson_xq2map(kin_info):
```

Because what it gets is a `kinematics information` object and not a dictionary.